automation/devops_automation_infra/k8s_plugins/kafka.py

```diff
@@ -71,7 +71,7 @@ def _expose(self):
 
         logging.debug("Waiting for kafka brokers to restart")
         waiter.wait_for_predicate(lambda: self._kafka_brokers_restarted(pods_timestamps) is True, timeout=30)
-        waiter.wait_for_predicate(lambda: self._is_running is True, timeout=60)
+        waiter.wait_for_predicate(lambda: self._is_running is True, timeout=120)
```

Comment:
why you increased the timeout? what changed?

Reply:
nothing changed but a couple of times in a row my test felt here and after 2 minutes the is running function returned true so I guess 60 sec wont make a difference, if it fails it fails

Reply:
Ok, pay attention please that if you run not on a local machine but on an aws machine, the timeout is sufficient for this case